endpoints/citrus-websocket/src/main/java/com/consol/citrus/websocket/client/WebSocketClientEndpointConfiguration.java

```diff
@@ -75,7 +73,7 @@ private CitrusWebSocketHandler getWebSocketClientHandler(String url) {
 
         if (webSocketHttpHeaders == null) {
             webSocketHttpHeaders = new WebSocketHttpHeaders();
-            webSocketHttpHeaders.setSecWebSocketExtensions(Collections.singletonList(new StandardToWebSocketExtensionAdapter(new JsrExtension(new PerMessageDeflateExtension().getName()))));
+            // webSocketHttpHeaders.setSecWebSocketExtensions(Collections.singletonList(new StandardToWebSocketExtensionAdapter(new JsrExtension(new PerMessageDeflateExtension().getName()))));
```

Comment:
didn't exactly see if `jsr356` is the default now? i think so. but since I've no clue what this is, I could not give detailed opinion.